bcManager/bcManager.py

```diff
@@ -1934,12 +1976,15 @@ def is_captain(self, player):
                 return True
         return False
     
-    def get_latest_account_replay_by_plat_id(self, guild, platform, plat_id):        
+    async def get_latest_account_replay_by_plat_id(self, guild, platform, plat_id):        
+        """ Get most recent replay by Platnium ID """
         bapi : ballchasing.Api = self.ballchasing_api[guild]
         data = bapi.get_replays(player_id=f"{platform}:{plat_id}", sort_by="replay-date", sort_dir="desc", count=1)
-        return next(data, {})
+        async for r in data:
+            return r # Workaround `StopAsyncIteration` exception with `__anext__()` for one replay
+        return None
 
-    def get_player_data_from_replay_by_plat_id(self, replay_json, platform, platform_id):
+    async def get_player_data_from_replay_by_plat_id(self, replay_json, platform, platform_id):
```

Comment:
why is this async?

Reply:
It's called in other async code and uses an async library now. It should have always been async.